samples/LibVLCSharp.Avalonia.Sample/ViewModels/MainWindowViewModel.cs

```diff
@@ -1,24 +1,42 @@
 ﻿using System;
+using Avalonia.Controls;
 using LibVLCSharp.Shared;
 
 namespace LibVLCSharp.Avalonia.Sample.ViewModels
 {
     public class MainWindowViewModel : ViewModelBase, IDisposable
     {
         private readonly LibVLC _libVlc = new LibVLC();
+        
+        public MediaPlayer MediaPlayer { get; }
+        
         public MainWindowViewModel()
         {
             MediaPlayer = new MediaPlayer(_libVlc);
+            Play();
         }
 
         public void Play()
         {
+            if (Design.IsDesignMode)
+            {
+                return;
+            }
+            
             using var media = new Media(_libVlc, new Uri("http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"));
             MediaPlayer.Play(media);
         }
-
-        public MediaPlayer MediaPlayer { get; }
-
+        
+        public void Stop()
+        {
+            if (!MediaPlayer.IsPlaying)
```

Comment:
you can remove that `if` I think, since paused != stopped and I try to keep the sample minimal. Just redirect the call to the mediaplayer directly :)

Reply:
Removed